aws/cloudformation/lambdas/sql-user/index.js

```diff
@@ -0,0 +1,130 @@
+const mysql = require("mysql");
+const { sendCfnResponse } = require("./cfn-response");
+
+exports.handler = async (event, context) => {
+  // TODO: Mask password fields.
+  console.log("REQUEST RECEIVED:\n", JSON.stringify(event));
+
+  const props = event.ResourceProperties;
+  const {
+    Name: name,
+    ClientHost: clientHost,
+    Password: password,
+    Databases: databases,
+    Privileges: privileges,
+  } = props;
+
+  let physicalResourceId = event.PhysicalResourceId || name;
```

Comment:
For clarity, I would suggest NOT destructuring/renaming the ResourceProperties fields. Shorten `event.ResourceProperties` to `props` helps readability, but creating a bunch of consts is less clear later on in the function. Reminding the reader that your using something from `props` removes any ambiguity about what it is.

```suggestion
  const props = event.ResourceProperties;
  
  let physicalResourceId = event.PhysicalResourceId || props.Name;
```